examples/src/main/scala/caliban/play/ExampleApp.scala

```diff
@@ -0,0 +1,45 @@
+package caliban.play
+
+import play.api.Mode
+import play.api.mvc.{DefaultControllerComponents, Handler, RequestHeader}
+import play.api.routing.sird._
+import play.core.server.AkkaHttpServer
+import play.core.server.ServerConfig
+
+import scala.io.StdIn.readLine
+
+object ExampleApp extends App {
+
+  val server = AkkaHttpServer.fromRouterWithComponents(
+    ServerConfig(
+      mode = Mode.Dev, 
+      port = Some(8088),
+      address = "127.0.0.1"
+    )
+  ) { components =>
+    val controller = new CalibanController(
+      DefaultControllerComponents(
+        components.defaultActionBuilder,
+        components.playBodyParsers,
+        components.messagesApi,
+        components.langs,
+        components.fileMimeTypes,
+        components.executionContext
+      )
+    )(components.actorSystem, components.materializer)
+    val router: PartialFunction[RequestHeader, Handler] = {
```

Comment:
Is it possible to serve graphiql from the resources like the other examples?

Reply:
Sure! done :)